google/cloud/security/common/gcp_api/compute.py

```diff
@@ -616,3 +640,39 @@ def get_instance_group_managers(self, project_id):
             project_id)
         return _flatten_aggregated_list_results(project_id, paged_results,
                                                 'instanceGroupManagers')
+
+    def get_project(self, project_id):
+        """Returns the specified Project resource.
+
+        Args:
+            project_id (str): The project id.
+
+        Returns:
+            dict: A Compute Project resource dict.
+            https://cloud.google.com/compute/docs/reference/latest/projects/get
+        """
+        try:
+            return self.repository.projects.get(project_id)
+        except (errors.HttpError, HttpLib2Error) as e:
+            api_not_enabled, details = _api_not_enabled(e)
+            if api_not_enabled:
+                raise api_errors.ApiNotEnabledError(details, e)
+            raise api_errors.ApiExecutionError(project_id, e)
+
+    def test_api_enabled(self, project_id):
```

Comment:
nit: the prefix ```test_``` seems too similar to actual unit test naming; how about replacing with standard boolean naming  ```is_``` prefix, such as ```is_api_enabled```?

Reply:
Good call, done.